src/main/java/com/gojek/beast/sink/bq/handler/impl/OOBErrorHandler.java

```diff
@@ -55,11 +58,12 @@ public Status handleErrorRecords(Map<Record, List<BQInsertionRecordsErrorType>>
                 recordsToWrite.add(record);
             }
         }
+        statsClient.gauge("data.error.records,type=oob", oobErrorRecordsCount);
         if (shouldBatchFail) {
             //lets not store OOB records as well as the batch contains invalid/schema related errors
             log.info("Batch with records size: {} contains invalid records, marking this batch to fail", records.size());
-            statsClient.gauge("sink.unprocessed.invalid.err.records", invalidRecordsCount);
-            statsClient.gauge("sink.unprocessed.unknown.err.records", unKnownErrorRecordsCount);
+            statsClient.gauge("data.error.records,type=invalid", invalidRecordsCount);
```

Comment:
Let us have this outside the if block, lets send 0 if the count is 0 rather than not sending this metric